checkstyle/suppressions.xml

```diff
@@ -30,7 +30,8 @@
               files="SerializerHook\.java$"/>
 
     <!-- ConsoleApp -->
-    <suppress checks="MethodCountCheck|FileLengthCheck|ClassFanOutComplexityCheck" files="com/hazelcast/console/ConsoleApp"/>
+    <suppress checks="MethodCountCheck|FileLengthCheck|ClassFanOutComplexityCheck"
```

Comment:
We should not be adding more suppressions to `suppressions.xml`. Could you please move these to the Java source code file, as a class annotation `@SuppressWarnings("checkstyle:methodcount")` etc.? Except for `FileLength`, `IllegalImport` and others which are beyond the class scope.